lib/reply.js

```diff
@@ -80,7 +80,8 @@ Object.defineProperty(Reply.prototype, 'sent', {
 
     this[kReplySentOverwritten] = true
     this[kReplySent] = true
-  }
+  },
+  configurable: true
```

Comment:
Help me understand. This property has both a getter and a setter. That should negate the need for `configurable: true`. Why are you setting this option?

Reply:
That's not correct. The `configurable` property dictates if the property can be deleted. Run this:

```js
'use strict'

const foo = {}
Object.defineProperty(foo, 'bar', {
  enumerable: true,
  get () {
    return this[Symbol.for('bar')]
  },
  set(value) {
    console.log('setting bar')
    this[Symbol.for('bar')] = value
  }
})

console.log(`foo = ${JSON.stringify(foo)}`)
foo.bar = 'baz'
console.log(`foo = ${JSON.stringify(foo)}`)
```

Notice that it works as expected. Now run this:

```js
'use strict'

function Foo () {}

Object.defineProperty(Foo.prototype, 'bar', {
  enumerable: true,
  get() {
    return this[Symbol.for('bar')]
  },
  set(value) {
    console.log('setting bar')
    this[Symbol.for('bar')] = value
  }
})

const foo = new Foo()
console.log(`foo = ${JSON.stringify(foo)}`)
foo.bar = 'baz'
console.log(`foo = ${JSON.stringify(foo)}`)
console.log(`foo.bar = ${foo.bar}`)
```

Notice that both of the stringify lines output `{}`, but the last one outputs `foo.bar = baz`. The property _is_ being modified. It is just being modified on the prototype, not the instance.

Reply:
For more fun with properties, study https://gist.github.com/jsumners/0a586bf6c3dcb835d692251347a0e730

Reply:
😣 oh javascript

Reply:
If the function is run but the report does not show it up, then let's just flag it as ignored by istanbul.